### ISSUE TITLE
Might work better with a different meta-optimizer - enhancement

### ISSUE THREAD

**Issue by AjayTalati** [ikostrikov/pytorch-meta-optimizer#1]
Hi @ikostrikov,

I just saw this, [Learning Gradient Descent: Better Generalization and Longer Horizons](https://arxiv.org/abs/1703.03633), and their TF repo, [rnnprop](https://github.com/vfleaking/rnnpropurl)

They claim their meta-optimizer works better than deepminds? Just thought it's worth posting here, in case anyone want's to look into it, as an enhancement to this repo?

Cheers,

Aj
